packages/core/src/interfaces/signer.ts

```diff
@@ -0,0 +1,6 @@
+import {Address} from "abitype";
+
+export interface SmartAccountSigner {
```

Comment:
the rest of the repo follows the pattern of having these live in a `types.ts` file in their namespace, so would probs put this in `signer/types.ts` alongside the `hd-account`